test/integration/shoots/operations/containerruntime.go

```diff
@@ -54,6 +54,9 @@ var _ = ginkgo.Describe("Shoot container runtime testing", func() {
 			Name:              extensionsv1alpha1.CRINameContainerD,
 			ContainerRuntimes: nil,
 		}
+		// containerD is supported only with Ubuntu OS for now.
+		// TODO: adapt/remove this when containerD is available on other OS.
+		containerdWorker.Machine.Image.Name = "ubuntu"
```

Comment:
I think we also have to set the machine.image.versions to nil as we are copying the existing worker pool. This way we can take advantage of the machine image version defaulting.

Reply:
Nice catch! I force-pushed.